src/Pages/ProductListPage.js

```diff
@@ -113,28 +81,30 @@ const ProductListPage = () => {
                             <input
                                 type="reset"
                                 value="Clear filters"
-                                onClick={resetFilters}/>
+                                onClick={()=>resetFilters()}/>
                         : null
                     }
                 </CategoriesContainer>
                 <ProductContainer>
                     {  
-                        hasTimePassed ?
-                            filteredProducts.products.map(item => {
+                        !areProductsLoaded ?
+                            filteredProducts.products.map((item,index) => {
                                 return(
-                                <li key={item.id}>
+                                <li key={index}>
                                     <ProductElement showCategory {...item} />
                                 </li>
                                 )
                             })
-                        : filteredProducts.products.map(item => {
+                        : filteredProducts.products.map( (item, index) => {
                             return(
-                                <ProductSkeleton key={item.id}/>
+                                <li key={index}>
+                                    <ProductSkeleton/>
+                                </li>
```

Comment:
The logic here is quite the same, the different is the component that gets rendered. I think this would be better written if you move your `areProductsLoaded` ternary inside and conditionally render one component or the other.